docs/markdown/index.md

```diff
@@ -33,19 +33,15 @@ developers. The first one is the mailing list, which is hosted at
 The second way is via IRC. The channel to use is `#mesonbuild` at
 [Freenode](https://freenode.net/).
 
-### Projects using Meson
+### [Projects using Meson](http://mesonbuild.com/Users.html)
```

Comment:
A bit late I know, but it's preferable to link this way: `Projects using Meson](Users.md)` , as it will correctly link to the local version of the page :)